analyzer/src/main/kotlin/managers/Unmanaged.kt

```diff
@@ -50,17 +50,18 @@ class Unmanaged(analyzerConfig: AnalyzerConfiguration, repoConfig: RepositoryCon
      * @param definitionFile The directory containing the unmanaged project.
      */
     override fun resolveDependencies(definitionFile: File): ProjectAnalyzerResult? {
+        val vcsInfo = VersionControlSystem.getCloneInfo(definitionFile)
         val project = Project(
                 id = Identifier(
                         type = toString(),
                         namespace = "",
-                        name = definitionFile.name,
-                        version = ""
+                        name = vcsInfo.url.split('/').last().removeSuffix(".git"),
```

Comment:
How about also using the `path` which is the `tag` in case of SVN and the `manifest` in case of `Git Repo`.?

Reply:
hm, thinking about it again even if not perfect the most important part seems to be setting the revision
and as SHA1 are unlikely to conflict even accross repos imo this is good enough for now.

Reply:
Actually no - the path needs to be added to ensure that the cache keys are not shared accross different projects / manifests.

Reply:
Done.

Reply:
Thx!